apps/api_web/lib/api_web/api_view_helpers.ex

```diff
@@ -240,13 +240,50 @@ defmodule ApiWeb.ApiViewHelpers do
     |> String.replace("/", "%2F")
   end
 
+  def default_registered_per_interval() do
+    ApiWeb.RateLimiter.max_registered_per_interval()
+  end
+
   def limit(%ApiAccounts.Key{} = key) do
     key
     |> ApiWeb.User.from_key()
     |> ApiWeb.RateLimiter.max_requests()
     |> trunc()
   end
 
+  def limit_value(%ApiAccounts.Key{} = key) do
```

Comment:
naming is hard, feel free to suggest a better name. this refers to the stored value of limit (as daily_value) in the database.